doozer/doozerlib/runtime.py

```diff
@@ -774,6 +774,18 @@ def get_global_arches(self):
         """
         return list(self.arches)
 
+    def get_global_konflux_arches(self):
+        """
+        :return: Returns a list of architectures that are enabled globally in group.yml, for konflux.
+        """
+        # For now, cli override (LIMIT_ARCHES) and arches_override in group config are not supported
+        arches = list(self.group_config.konflux_arches)
```

Comment:
Suggesting to add a new key `konflux` for all konflux-specific options and move `arches` under `konflux`. e.g.

```yaml
konflux:
  arches: ['x86_64']
```
Reasons:
1. We already use this model in image metadata config, e.g. https://github.com/openshift-eng/ocp-build-data/blob/5436cfe344a48898be6a3fcb32a9bb0d046082d2/images/driver-toolkit.yml#L41. This will look more consistent.
2. In the future when we add more konflux-specific options, we can use a similar approach to merge options under `konflux` to global group options.